File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/model/redis/ValueVo.java
package org.javamaster.invocationlab.admin.model.redis;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

/**
 * @author yudong
 */
@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class ValueVo {
    private String redisValue;
    private Integer redisValueSize;
    private Boolean redisValueJdkSerialize;
    private Class<?> redisValueClazz;
    private String redisKeyBase64;

    private Long redisKeyTtl;
    private String redisKey;
    private Class<?> redisKeyClazz;
    private Boolean redisKeyJdkSerialize;
    private String redisKeyType;

    private List<FieldVo> fieldVos;
    private Integer fieldCount;
}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/model/redis/ValueVo.kt
package org.javamaster.invocationlab.admin.model.redis

import org.javamaster.invocationlab.admin.annos.AllOpen

/**
 * @author yudong
 */


@AllOpen
class ValueVo {
    var redisValue: String? = null
    var redisValueSize: Int? = null
    var redisValueJdkSerialize: Boolean? = null
    var redisValueClazz: Class<*>? = null
    var redisKeyBase64: String? = null

    var redisKeyTtl: Long? = null
    var redisKey: String? = null
    var redisKeyClazz: Class<*>? = null
    var redisKeyJdkSerialize: Boolean? = null
    var redisKeyType: String? = null

    var fieldVos: List<FieldVo>? = null
    var fieldCount: Int? = null
}
